cartographer/io/serialization_format_migration_test.cc

```diff
@@ -100,6 +102,63 @@ class MigrationTest : public ::testing::Test {
   static constexpr int kNumOriginalMessages = 9;
 };
 
+class SubmapHistogramMigrationTest : public ::testing::Test {
+ protected:
+  void SetUp() {
+    CreateNode();
+    CreateSubmap();
+    CreatePoseGraphWithNodeToSubmapConstraint();
+  }
+
+ private:
+  void CreateSubmap() {
```

Comment:
Can you use CreateFakeSubmap3D() from mapping/internal/testing/test_helpers.h instead of adding this?
There is also CreateFakeNode(). Or is that header internal to mapping/*?

Reply:
Thanks for this hint! This allowed me to remove most of this code.